### PR TITLE
Fix nuget pipeline

### PR DESCRIPTION
**Pull request by Saadnajmi**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

The Nuget Package was overpacking (packing every file), including files with a "." in front of them which made NuGet unhappy. This caused our publish PR to fail (luckily only after it published the npm packages). 
To fix the pipeline, let's just specify some hardcoded paths of files to specify so it's not packing everything.

### Verification

I have a test pipeline [here](https://dev.azure.com/ms/ui-fabric-react-native/_build/results?buildId=111747&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=b6761d2c-04f1-5cc8-8d6b-e444be405a7a) that ran the changes (with the publish npm packages step commented out so that I don't accidentally mess with that).

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: .ado/azure-pipelines.publish.yml [hunk 1]
```diff
@@ -28,11 +28,11 @@ steps:
...
-    displayName: 'yarn test'
-  - script: |
-      yarn publish:beachball -- -b origin/master -n $(npmAuth) --access public -y
-    displayName: 'Publish NPM Packages'
+  # - script: |
+  #     yarn test
+  #   displayName: 'yarn test'
+  # - script: |
+  #     yarn publish:beachball -- -b origin/master -n $(npmAuth) --access public -y
+  #   displayName: 'Publish NPM Packages'
```
Comment (Saadnajmi) [inline .ado/azure-pipelines.publish.yml:36]: Had this commentd out so I could test without messing with the npm packages. Reverting now that I have

File: package.nuspec [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        <id>Microsoft.FluentUI.ReactNative</id>
+        <version>$buildNumber$</version>
+        <description>A react-native component library that implements the Fluent Design System.</description>
+        <authors>Microsoft</authors>
+        <projectUrl>https://github.com/microsoft/fluentui-react-native.git</projectUrl>
+        <repository type="git" url="$repoUri$" commit="$commitId$"/>
+    </metadata>
+    <files>
+        <file src="packages/**/*.*" />
+        <file src="scripts/**/*.*" />
```
Comment (tkamireh) [inline package.nuspec:13]: Out of curiosity, what do you need from scripts?

**Review by tkamireh — APPROVED** [review]
one question, otherwise lgtm

**Review by HeyImChris — APPROVED** [review]

File: package.nuspec (hunk 2 above)
Comment (Saadnajmi) [reply package.nuspec:13]: Probably nothing. I realize that the code wasn't bundled beforehand and so the nuget package as is might need scripts to help with bundling. 

When I'm not oof, I'll probably investigate having the pipeline bundle / build native apple libraries and pack all of those. That would also mean the package would become apple specific. 😅. Another day though

**Pull request merged**